code/obj/machinery/computer/bank_records.dm

```diff
@@ -13,6 +13,25 @@
 	var/printing = null
 	var/can_change_id = 0
 	var/payroll_rate_limit_time = 0 //for preventing coammand message spam
+	var/static/bonus_rate_limit_time = 0 //prevent bonus spam because these have an annoucement
+	///I know we already have department job lists but they suck and are brittle and way too general so I made my own here
+	var/static/list/departments = list(
+		"Stationwide" = list(/datum/job/), //hacky, yet generic
```

Comment:
Oough this will iterate every job datum in the codebase for every player on the server. It's fine when it's more restricted but I'd rather this was specialcased to skip the loop instead.

Reply:
Oh sure i'll just catch it

Reply:
Oh sure i'll just catch it